gradle.properties

```diff
@@ -2,14 +2,15 @@ org.gradle.caching=true
 group=org.ballerinax.azure.servicebus
 version=3.7.0
 
-puppycrawlCheckstyleVersion=10.12.1
-checkstyleToolVersion=10.12.1
-githubSpotbugsVersion=5.0.14
-githubJohnrengelmanShadowVersion=8.1.1
-underCouchDownloadVersion=5.4.0
-researchgateReleaseVersion=2.8.0
-testngVersion=7.6.1
+checkstylePluginVersion=10.12.0
+spotbugsPluginVersion=5.0.14
+shadowJarPluginVersion=8.1.1
+downloadPluginVersion=5.4.0
+releasePluginVersion=2.8.0
+ballerinaGradlePluginVersion=2.1.5
+jacocoVersion=0.8.10
 
 ballerinaLangVersion=2201.8.0-20230830-220400-8a7556d8
 azureServiceBusVersion=7.14.3
 slf4jVersion=1.7.30
+jacksonVersion=2.15.2
```

Comment:
Where do we use this jackson dependency?

Reply:
My mistake. Addressed in [4ae71d8](https://github.com/ballerina-platform/module-ballerinax-azure-service-bus/pull/205/commits/4ae71d81c47f3fb1ef327b86f3c4ff4cf43c4e83)